lib/rpc/proto/discovery.proto

```diff
@@ -16,6 +16,15 @@ service Discovery {
 
     // GetCurrentTime returns the node's current time as UTC timestamp
     rpc GetCurrentTime(google.protobuf.Empty) returns (google.protobuf.Timestamp);
+
+    // GetVersion returns the agent's version information.
+    rpc GetVersion(google.protobuf.Empty) returns (Version);
+}
+
+// Version describes an agent's version information
+message Version {
+    // Payload is the version information payload.
+    bytes payload = 1;
```

Comment:
I would rather have the protobuf to be the source of truth for data types.

Reply:
Moved Version definition from modules to proto pkg. a583350